frontend/public/components/terminal.jsx

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Terminal as XTerminal } from 'xterm';
-import * as fit from 'xterm/lib/addons/fit/fit';
-import * as full from 'xterm/lib/addons/fullscreen/fullscreen';
+import { FitAddon } from 'xterm-addon-fit';
+import { withTranslation } from 'react-i18next';
 import { CompressIcon } from '@patternfly/react-icons';
 import { Button } from '@patternfly/react-core';
-import { withTranslation } from 'react-i18next';
-
-XTerminal.applyAddon(fit);
-XTerminal.applyAddon(full);
+import { XtermAddonFullscreen } from '@console/shared';
```

Comment:
Why do we need custom implementation of fullscreen addon? Did xterm remove support for their fullscreen addon?

Reply:
iirc @sahil143 said they removed this addon. I'm not sure if they replaced or why it was removed though.

Reply:
Yeah, seems like they removed fullscreen addon - https://github.com/xtermjs/xterm.js/pull/2065#issuecomment-491614794